Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedURL.java

```diff
@@ -42,7 +47,16 @@ public boolean getIsRead() {
     return isRead;
   }
 
+  public String getTitle() {
+    return crawler.getTitle();
+  }
+
+  public String getContent() {
+    return crawler.getContent();
+  }
+
   public int getWordCount() {
+    wordCount = crawler.getWordCount();
```

Comment:
Looks like the variable `wordCount` is not referred anywhere. What's the benefit to store the value here?